pkg/controller/migmigration/migrate.go

```diff
@@ -92,15 +91,12 @@ func (r *ReconcileMigMigration) migrate(migration *migapi.MigMigration) (time.Du
 		return NoReQ, nil
 	}
 
-	// Running
-	step, n, total := task.Itinerary.progressReport(task.Phase)
-	message := fmt.Sprintf(RunningMessage, n, total)
 	migration.Status.SetCondition(migapi.Condition{
 		Type:     Running,
 		Status:   True,
-		Reason:   step,
+		Reason:   task.Step,
 		Category: Advisory,
-		Message:  message,
+		Message:  PhaseDescriptions[task.Phase],
```

Comment:
We need to verify that a PhaseDescription for the Phase exists before setting it here.